raven/src/main/java/com/getsentry/raven/Raven.java

```diff
@@ -23,6 +24,9 @@
  */
 public class Raven {
     private static final Logger logger = LoggerFactory.getLogger(Raven.class);
+    // CHECKSTYLE.OFF: ConstantName
+    private static final Logger lockdownLogger = LoggerFactory.getLogger(Raven.class + ".lockdown");
+    // CHECKSTYLE.ON: ConstantName
```

Comment:
Checkstyle only allows for `logger`, I could also change the checkstyle rules themselves...